Directory.build.props

```diff
@@ -4,24 +4,25 @@
     <PackageLicenseExpression>MS-PL</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/MvvmCross/MvvmCross</PackageProjectUrl>
     <PackageIconUrl>http://i.imgur.com/Baucn8c.png</PackageIconUrl>

```

Comment:
We should probably change this to PackageIcon too. As PackageIconUrl is deprecated

Reply:
PackageIcon is not working on older nuget clients. I've added the code for it, but disabled for now. Maybe we wait a bit longer to enable it?